packages/@divvi/mobile/src/earn/EarnDepositConfirmationScreen.tsx

```diff
@@ -48,12 +54,22 @@ import { useTokenInfo, useTokenToLocalAmount } from 'src/tokens/hooks'
 import { feeCurrenciesSelector } from 'src/tokens/selectors'
 import type { TokenBalance } from 'src/tokens/slice'
 import { getTokenBalance } from 'src/tokens/utils'
-import { getPreparedTransactionsPossible } from 'src/viem/preparedTransactionSerialization'
+import Logger from 'src/utils/Logger'
+import {
+  getPreparedTransactionsPossible,
+  getSerializablePreparedTransactions,
+} from 'src/viem/preparedTransactionSerialization'
 import {
   getFeeCurrencyAndAmounts,
   type PreparedTransactionsPossible,
 } from 'src/viem/prepareTransactions'
 
+const TAG = 'send/EarnDepositConfirmationScreen'
+const APP_TERMS_AND_CONDITIONS_URL = 'https://valora.xyz/terms'
```

Comment:
I know this was in the existing code, but it's incorrect and it should use the value we already have in the Statsig app config.

https://github.com/divvixyz/divvi-mobile/blob/f1f5668439412a557b40443b4069d9b73911f8b2/packages/@divvi/mobile/src/statsig/constants.ts#L116

Can be done in a separate PR.